new-arch-migration/EMBEDDED.md

```diff
@@ -0,0 +1,211 @@
+# In-place migration with embedded databases
+
+## Requirements
+
+GitGuardian provides a set of scripts that require specific tools to be installed on your host to facilitate application migration:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) (version ≥ 1.27.0)
+- [kubectl kots plugin](https://docs.replicated.com/reference/kots-cli-getting-started#install) (version ≥ 1.107.7)
+- [yq](https://mikefarah.gitbook.io/yq/) (Only for Blue/Green Migration)
+
+You need to be an administrator of the GitGuardian namespace where the application is deployed.
+
+The new version must use the same GitGuardian version as the legacy version. Please ensure you have the latest legacy version installed before upgrading to the new version.
```

Comment:
This sentence looks a bit confusing: **version** refers to both
- the architecture 
- the self-hosted release.

Maybe use the **architecture** word?